drivers/hyperv/hyperv_windows.go

```
@@ -334,7 +335,13 @@ func (d *Driver) Start() error {
 	if err != nil {
 		return err
 	}
-	return d.wait()
+
+	if err := d.wait(); err != nil {
+		return err
+	}
+
+	d.IPAddress, _ = d.GetIP()
```

Why swallow the error?